infra/service.tf

```diff
@@ -54,6 +54,10 @@ resource "google_cloud_run_v2_service" "server" {
         name  = "OTEL_PYTHON_EXCLUDED_URLS"
         value = "healthy"
       }
+      env {
```

Comment:
suggestion: Add a comment with additional context on the deployment suffix. When is it required/what's it for?